en/faq.md

```diff
@@ -0,0 +1,77 @@
+## FAQ
```

Comment:
Better for this to be in a folder, allowing consistent linking approach. Could be "general" or whatever.

Reply:
I moved it to last.